### PR TITLE
FAQ: Add another problem to be aware of when co-locating a repo

### PR DESCRIPTION
**Pull request by ilyagr**
Cc @jennings 

# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (cli/src/config-schema.json)
- [ ] I have added tests to cover my changes

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -55,6 +55,13 @@ important ones are:
 - Interleaving `git` and `jj` commands may create confusing branch conflicts or
   divergent changes.
 
+- If the working copy commit or its parent contain any conflicted files, tools
+  expecting a Git repo will interpret the commit contents and/or diff in a wrong
+  and confusing way. You should avoid doing mutating operations with Git tools
+  and ignore the confsuing information such tools present for conflicted commits
```
Comment (PhilipMetzger) [inline docs/FAQ.md:61]: Nit: Typo "confsuing" => "confusing"

**Review by PhilipMetzger — APPROVED** [review]
LG, maybe add a note to follow the bug about a friendlier format here.

File: docs/FAQ.md (hunk 1 above)
Comment (ilyagr) [reply docs/FAQ.md]: Thanks!

**Comment by ilyagr** [comment]
Thank you, Philip! I added a reference to https://github.com/martinvonz/jj/issues/3979, is that the bug you meant?

**Pull request merged**